guides/release/accessibility/components.md

```diff
@@ -1,89 +1,88 @@
-When crafting an accessible component, the first and most important thing is that the component should render valid HTML. 
+Lorsque que vous créez un composant accessible, la première chose et la plus importante de toute est que le composant rende un HTML valide.
 
-Both the HTML and ARIA specifications have been written in a way that make them work together. Semantic HTML provides the necessary _context_ to screen readers.
+Le HTML et les spécifications ARIA ont été écrit de manière à ce qu'ils fonctionnent ensembles. La sémantique HTML propose le contexte _nécessaire_ pour les lecteurs d'écrans.
```

Comment:
```suggestion
Les spécifications HTML et ARIA ont été rédigées de manière à fonctionner ensemble. La sémantique HTML propose le contexte _nécessaire_ pour les lecteurs d'écrans.
```

Reply:
It's "**context**" that is underlined in English, probably because it's used also as a tech word. We can keep the underline or drop it if its meaning is confusing, as you like.